R/pkg/vignettes/sparkr-vignettes.Rmd

```diff
@@ -27,6 +27,9 @@ library(SparkR)
 
 We use default settings in which it runs in local mode. It auto downloads Spark package in the background if no previous installation is found. For more details about setup, see [Spark Session](#SetupSparkSession).
 
+```{r, include=FALSE}
+SparkR:::sparkCheckInstall()
```

Comment:
Is the Rmd file a part of the install that the users see ? I just dont want to put in any code that people might copy-paste etc. Is it not good enough to pass in `master=local[*]` here ?